commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryLocalServiceImpl.java

```diff
@@ -648,4 +724,7 @@ private long _getFileEntryId(
 	private static final Log _log = LogFactoryUtil.getLog(
 		CPAttachmentFileEntryLocalServiceImpl.class);
 
+	@ServiceReference(type = AssetCategoryLocalService.class)
+	private AssetCategoryLocalService _assetCategoryLocalService;
```

Comment:
I think we should actually generate this injection using a `<reference>` tag in the service.xml (which would inject the service into the `CPAttachmentFileEntryLocalServiceBaseImpl`, iirc).